src/runtime/micro/host_low_level_device.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ *  Copyright (c) 2019 by Contributors
+ * \file host_low_level_device.h
+ * \brief emulated low-level micro device implementation on host machine
+ */
+#ifndef TVM_RUNTIME_MICRO_HOST_LOW_LEVEL_DEVICE_H_
+#define TVM_RUNTIME_MICRO_HOST_LOW_LEVEL_DEVICE_H_
+
+#include <sys/mman.h>
+#include <cstring>
+#include <memory>
+#include "low_level_device.h"
+#include "micro_common.h"
+
+namespace tvm {
+namespace runtime {
+/*!
+ * \brief emulated low-level device on host machine
+ */
```

Comment:
We do not need to expose HostLowlevelDevice into a header. So to minimize the number of .h files. Consider directly move it into the implementation (host_low_level_device.cc).

Move the declaration of HostLowLevelDeviceCreate into low_level_device.h